kike-canaries/canairio_firmware
# OTA via internet (remote updates)

- [x] publication on bintray from env variables and build flavor (tested)
- [x] Alternative to Bintray down service, ready via fota lib
- [x] Server side implementation via lighthttpd
- [x] Fixed issue with FOTA headers  
- [x] check for a new version on wifi loop
- [x] millis() issue for OTA interval 
- [x] multiple firmware flavor flow
- [x] restored CanAirIO sensor lib dependencies and stable target
- [x] restored `build all` script parameters 
- [x] tested on all boards

To road map:

- check watchdog issue on slow connections
- check device id on header request
- refactor pre-scripts on all flavors
